ds701_book/03-Probability-and-Statistics-Refresher.qmd

```diff
@@ -296,7 +296,7 @@ Now if $B$ is a proper subset of $\Omega,$ then $P(B) < 1$.   So $P(\cdot|B)$ is
 :::
 ::: {.column width="40%"}
 
-![](drawio/ConditionalProbability.png){fig-align="center" width=80%}
+![](figs/Lec03-ConditionalProbability.png){fig-align="center" width=80%}
```

Comment:
would be nice to indicate somehow that these are drawio editable pngs. perhaps add '-drawio' to the end of the name?

Reply:
I'm reverting this back to the drawio folder. Then all drawio files are there and it is clear they are editable pngs.